lmdeploy/model.py

```diff
@@ -43,6 +36,7 @@ class ChatTemplateConfig:
     eoh: Optional[str] = None
     assistant: Optional[str] = None
     eoa: Optional[str] = None
+    stop_word_suffix: Optional[str] = None
```

Comment:
why `suffix`?

Reply:
For interactive prompts. For example, `eoa='<eoa>'`, `stop_word_suffix='\n'`

Reply:
The name is confusing. Are you suggesting a separator after `eoa`?

Reply:
Yeah, the previous name is `eoa_suffix`. However, I found the following case:
```python
                 eoa='<|im_end|>',
                 stop_word_suffix='\n',
                 stop_words=['<|im_end|>', '<|action_end|>'],
```
Shall I rename it back?

Reply:
How about a `separator`?